pkg/asset/agent/mirror/registriesconf.go

```diff
@@ -163,7 +174,7 @@ func (i *RegistriesConf) Generate(dependencies asset.Parents) error {
 	i.setMirrorConfig(i.Config)
 
 	if !i.releaseImageIsSameInRegistriesConf(releaseImage.PullSpec) {
-		logrus.Warnf(fmt.Sprintf("The ImageContentSources configuration in install-config.yaml should have at least one source field matching the releaseImage value %s", releaseImage.PullSpec))
+		logrus.Warnf(fmt.Sprintf("The ImageContentSources or ImageDigestMirrorSet configuration in install-config.yaml should have at least one source field matching the releaseImage value %s", releaseImage.PullSpec))
```

Comment:
Should be `imageDigestSources` as that's the field name in install-config.
I understand why ImageContentSources is still mentioned here, but if it is deprecated then I'd be tempted to stop encouraging users to put more things in it.